### PR TITLE
dexer: Change the project name & org to dexer and dgplug

### PR DESCRIPTION
**Pull request by sayanchowdhury**
Signed-off-by: Sayan Chowdhury <sayan.chowdhury2012@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## File Indexer [![Build Status](https://travis-ci.org/farhaanbukhsh/file-indexer.svg?branch=master)](https://travis-ci.org/farhaanbukhsh/file-indexer)
+## File Indexer [![Build Status](https://travis-ci.org/farhaanbukhsh/dexer.svg?branch=master)](https://travis-ci.org/farhaanbukhsh/dexer)
```
Comment (hellozee) [inline README.md]: Missed to change `farhaanbukhsh` to `dgplug`

**Comment by farhaanbukhsh** [comment]
can we do a grep and see if we still have "farhaanbukhsh" in the files?

**Comment by hellozee** [comment]
@farhaanbukhsh we better be doing that, :wink:

File: README.md (hunk 1 above)
Comment (sayanchowdhury) [reply README.md]: I left that knowingly. Seems like that also got upgraded.

**Comment by hellozee** [comment]
LGTM for now at least, thanks for the patch @sayanchowdhury

**Comment by jee1mr** [comment]
> can we do a grep and see if we still have "farhaanbukhsh" in the files?

Shouldn't you be 'dexer'ing it instead? :P

**Pull request merged**